serverless/DEVELOPER_README.md

```diff
@@ -280,7 +280,11 @@ The user **deploying** this project **must** have the following permissions list
 				"apigateway:PUT",
 				"apigateway:TagResource"
 			],
-			"Resource": "*"
+			"Resource": [
+                "arn:aws:apigateway:<region>::/apis/*",
```

Comment:
Can you fix the indentation so it aligns with the rest of the policy.

Reply:
Changed the spaces to tabs.